kernel/threads.c

```diff
@@ -143,3 +155,34 @@ void block_current_thread (void) {
   running_tcb->state = TS_BLOCKED;
   schedule();
 }
+
+int fork_ret = 0;
```

Comment:
Seeing as this is only used in threads-asm.s it should probably also be defined there, without global scoping.